drivers/display/ssd1306.c

```diff
@@ -20,25 +20,6 @@ LOG_MODULE_REGISTER(ssd1306, CONFIG_DISPLAY_LOG_LEVEL);
 
 #include "ssd1306_regs.h"
 
-#if DT_INST_PROP(0, segment_remap) == 1
-#define SSD1306_PANEL_SEGMENT_REMAP	true
-#else
-#define SSD1306_PANEL_SEGMENT_REMAP	false
-#endif
-
-#if DT_INST_PROP(0, com_invdir) == 1
-#define SSD1306_PANEL_COM_INVDIR	true
-#else
-#define SSD1306_PANEL_COM_INVDIR	false
-#endif
-
-#if DT_INST_PROP(0, com_sequential) == 1
-#define SSD1306_COM_PINS_HW_CONFIG	SSD1306_SET_PADS_HW_SEQUENTIAL
-#else
-#define SSD1306_COM_PINS_HW_CONFIG	SSD1306_SET_PADS_HW_ALTERNATIVE
-#endif
-
-#define SSD1306_PANEL_NUMOF_PAGES	(DT_INST_PROP(0, height) / 8)
```

Comment:
This patch should be first change to add multi instance support for this driver.

Reply:
I reordered commits.

Reply:
@soburi this is still the last patch, can you check again?

Reply:
@fabiobaltieri 
Thank you for pointing out. It's certainly not good.
I fixed it.